collector/impl/prometheus_metrics_collector.go

```diff
@@ -31,6 +31,9 @@ import (
 	"github.com/hawkular/hawkular-openshift-agent/http"
 	"github.com/hawkular/hawkular-openshift-agent/log"
 	"github.com/hawkular/hawkular-openshift-agent/prometheus"
+	"github.com/hawkular/hawkular-openshift-agent/util/expand"
+	"os"
+	"strings"
```

Comment:
The project use "gofmt" (i have my Eclipse IDE set to run "gofmt" after every save so I don't have to think about it). And if formatting using gofmt, that covers almost everything so styling checking isn't needed for virtually everything else. In this case, the import convention (putting standard imports at the top, third-party package in the middle, and hawkular imports last, with each section separated by a blank line) is a convention that is not enforced by gofmt so that is one that needs to be known by committers. But that is about the only code-syntax rule I can think of that isn't handled by gofmt. So I don't think we need to add a checkstyle-like plugin - gofmt pretty much gives us everything else. It's not a big deal.